docs/GOVERNANCE.md

```diff
@@ -159,51 +165,79 @@ The specifications produced will adhere to the following:
+  Git branches will be used as a way to snapshot/tag releases. While we hope
+  to not push new commits to these branches, in practice, it is possible that
+  changes to a release might be needed without "picking up" other changes that
+  might have been commited to "main". Creating the branch during the release
+  process will makes this easier to manage should the need should arise.
+
+- Determine the new release version string. It should be of the form:
+  `<subject>/vX.Y.Z`, e.g. `cloudevents/v1.0.4` or `subscriptions/v1.0.0`.
+
+- Create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string appended with
+    `-rc#` as appropriate. Make sure to remove all `-wip` suffixes as needed.
+  - Merge the PR.
+  - Initiate a final review/test of the release.
+
+- A "release candidate" tag will be created with the new release version
+  string but with a suffix of `-rc#` (release candidate). This will indicate
+  that the authors believe it is ready for final review/testing. This will be
+  true for updates to existing specifications and for new specifications.
+
+- When review/testing is completed, create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string (w/o `-rc#`) as
+    appropriate.
+  - Update [RELEASES.md](RELEASES.md) to mention the new release, and
+    reference the yet-to-be-created release tag.
+  - Update the appropriate `*/RELEASE_NOTES.md` file with the changes
+    for the release. The list can be generated via:
+    `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`
+    by replacing "v0.1" with the name of the previous release. Or, use github's
+    [new release](https://github.com/cloudevents/spec/releases/new) process
+    to generate the list w/o actually creating the release yet.
+
 - Merge the PR.
-- Create a [new release](https://github.com/cloudevents/spec/releases/new):
-  - Choose a "Tag version" of the form: `<subject>/vX.Y.Z`, e.g.
-    `cloudevents/v1.0.4` or `subscriptions/v1.0.0`
-  - Target should be `main`, the default value
-  - Release title should be the same as the Tag - `<subject>/vX.Y.Z`
-  - Add some descriptive text, or the list of PRs that have been merged since
-    the previous release. The git query to get the list commits since the last
-    release is:
-    `git log --pretty=format:%s main...v0.1 | grep -v "Merge pull"`.
-    Just replace "v0.1" with the name of the previous release.
-  - Press `Publish release` button
-- Create an "announcement" highlighting the key features of the new release and
-  any potential noteworthy activities of the group:
-  - Send it to the mailing list
-  - Announce the release on our
-    [twitter account](http://twitter.com/cloudeventsio)
+  - Note that the link checker should fail since any references to the new
+    release tag will not be valid yet. This is expected.
+
+- Create a new branch with the same name as the new release version string
```

Comment:
If we're keeping the tag up to date with the tip of the branch, do we actually need them to have different names? The purpose of that was primarily to disambiguate in URLs, right? (Maybe it's better to avoid the conflict anyway, but it's probably worth discussing.)

Reply:
Resolving - that's fine.